src/main/java/eu/hbp/mip/models/DTOs/MIPEngineTabularVisualizationDTO.java

```diff
@@ -0,0 +1,27 @@
+package eu.hbp.mip.models.DTOs;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.Setter;
+
+import java.util.HashMap;
+import java.util.List;
+
+@Getter
+@Setter
+@AllArgsConstructor
+public class MIPEngineTabularVisualizationDTO {
```

Comment:
This does not belong to MIPEngine but to portal-backend / frontend. I think no prefix is needed for that... Not sure though.